test/e2etest.py

```diff
@@ -2160,38 +2160,14 @@ def test_healthcheck():
 def test_stats():
     re = requests.get(base_url + 'stats')
     assert re.status_code == 200
-    data = re.json()
```

Comment:
Speaking of testing, are there any validation checks we can do on the data outside of just checking for HTTP 200? Something like what's deleted here, to make sure we're returning data that looks like it's correct. Doesn't need to be as extensive; spot checking is probably fine.